src/gameHandlerClass.ts

```diff
@@ -0,0 +1,42 @@
+import { Player } from "./playerClass";
+import { Pokemon } from "./pokemonClass";
+
+export class GameHandler {
+  constructor(private _playerOne: Player, private _playerTwo: Player) {}
+
+  _currentPlayer: Player = this._playerOne;
+
+  getCurrentPlayer(): Player {
+    return this._currentPlayer;
+  }
+
+  getOpponentPlayer(): Player {
+    if (this._currentPlayer === this._playerOne) return this._playerTwo;
+    return this._playerOne;
```

Comment:
You can flex your programming skills and astonish your peers by using ternary operator:
```suggestion
    return this._currentPlayer === this._playerOne ? this._playerTwo : this._playerOne;
```